src/InsaLan/TournamentBundle/Resources/views/Default/tournament.html.twig

```diff
@@ -27,7 +27,33 @@
   <header class="game">
     <div class="container">
       <img style="height:100px" src="{{ asset(t.getUploadDir() ~ t.logoPath) }}" alt="Logo"/>
-      {{ t.name }}
+      {{ t.name }} -
+      {% if t.tournamentClose | date("m") == "1" %}
+        Janvier
+      {% elseif t.tournamentClose | date("m") == "2" %}
+        Février
+      {% elseif t.tournamentClose | date("m") == "3" %}
+        Mars
+      {% elseif t.tournamentClose | date("m") == "4" %}
+        Avril
+      {% elseif t.tournamentClose | date("m") == "5" %}
+        Mai
+      {% elseif t.tournamentClose | date("m") == "6" %}
+        Juin
+      {% elseif t.tournamentClose | date("m") == "7" %}
+        Juillet
+      {% elseif t.tournamentClose | date("m") == "8" %}
+        Août
+      {% elseif t.tournamentClose | date("m") == "9" %}
+        Septembre
+      {% elseif t.tournamentClose | date("m") == "10" %}
+        Octobre
+      {% elseif t.tournamentClose | date("m") == "11" %}
+        Novembre
+      {% else %}
+        Décembre
+      {% endif %}
+      {{ t.tournamentClose | date("Y") }}
```

Comment:
I strongly suggest you to use the translation mechanism of Symfony instead of that. The doc is here https://symfony.com/doc/current/translation.html

Precisely here what you look for is that : https://twig-extensions.readthedocs.io/en/latest/
If you install the `twig/extensions` with composer, you can have all the previous lines reduced to `{{ t.tournamentClose | localizeddate('meduim', none, 'fr_FR') }}`

Reply:
We need to update `twig/extensions` for that.
Intl has been introduced in 1.2.0 and we are still in ~1.0

Reply:
@AntoineBvn according to https://getcomposer.org/doc/articles/versions.md#tilde-version-range- it does not seem to be a problem^^ (at least my install was using 1.5.2).

The extension has to be actived via config.yml https://coderwall.com/p/orvogq/symfony2-localized-date-string-in-twig-template

Reply:
My bad, I made a mistake about what means `~`.
Can you do it @FoxyHawk ?

Reply:
Patch is up ^^

I needed to install php-intl on my system and added twig extension intl as a service.